src/lib/sections/Navigation/Navigation.scss

```
@@ -168,14 +176,24 @@ $side-navigation-z-index: 103;
   }
 }
 
-.l-navigation, .l-navigation-bar {
+.p-panel__footer {
+  margin-top: auto;
```

I think we should remove the padding on content within the footer, as well as the divider pseudo-element

```suggestion
  margin-top: auto;
  
  .p-panel__content {
    padding: 0;
    
    .p-side-navigation__list:last-of-type::after {
      content: none;
    }
  }
```